src/makielayout/lineaxis.jl

```diff
@@ -674,9 +679,13 @@ function get_ticks(m::MultiplesTicks, any_scale, ::Automatic, vmin, vmax)
     dvmax = vmax / m.multiple
     multiples = Makie.get_tickvalues(LinearTicks(m.n_ideal), dvmin, dvmax)
 
-    multiples .* m.multiple, Showoff.showoff(multiples) .* m.suffix
+    multiples .* m.multiple, showoff_minus(multiples) .* m.suffix
 end
 
+# Replaces hyphens in negative numbers with the unicode MINUS_SIGN
+function showoff_minus(x::AbstractVector)
```

Comment:
Seems like quite the effort as well, especially considering that the tick code is pretty performance sensitive?
Do we memoise this somehow?

Reply:
yeah this one was a rather hacky way to get it working, as I couldn't access `showoff`'s logic directly..